scripts/exgdas_atmos_chgres_forenkf.sh

```diff
@@ -154,7 +154,7 @@ EOF
          export pgm=$CHGRESNCEXEC
          . prep_step
 
-         $APRUN_CHGRES $CHGRESNCEXEC chgres_nc_gauss0$FHR.nml
+         $APRUN_CHGRES $CHGRESNCEXEC chgres_nc_gauss0$FHR.nml && true
          export err=$?; err_chk
```

Comment:
Do we want to uncompress lines like this and make them two lines?

Reply:
No, it needs to stay as it is.  This isn't a logical test, but a way to get around a failed status when running with `set -e`.  It actually does nothing except prevent a non-zero return status from exiting the script.  This allows err_chk to be called in such cases.  Otherwise, the J-Job would just exit.

Reply:
We're talking about the `export err=$?; err_chk` line, right?

Reply:
Oh, sorry, I was looking at the line above.  Yes, we can make it two lines.

Reply:
Cool, thanks! I also noted the same on line 173.

Reply:
Fixed.